input/fsh/examples/rheumatic-fever/WaikatoRFSecondaryPreventionService.fsh

```diff
@@ -0,0 +1,18 @@
+Instance: WaikatoRFSecondaryPreventionService
+InstanceOf: NzOrganization
+Description: "Identifies the Waikato rheumatic fever secondary prevention provider organization"
+Usage: #example
+
+* meta.profile = "http://hl7.org.nz/fhir/StructureDefinition/NzOrganization"
+* meta.versionId = "1"
+* meta.lastUpdated = "2023-10-06T08:00:00Z"
+
+* identifier.use = #official
+* identifier.system = "https://standards.digital.health.nz/ns/hpi-organisation-id"
+* identifier.value = "GOM087-D"
+
+// * identifier[orgId].use = #official
```

Comment:
Might want to remove this commented code....

Reply:
done